python/sparkdl/graph/utils.py

```diff
@@ -0,0 +1,163 @@
+#
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import logging
+import six
+import webbrowser
+
+import tensorflow as tf
+
+logger = logging.getLogger('sparkdl')
+
+"""
```

Comment:
Utilities defined in this module are supposed to run as if they were part of TensorFlow utilities, e.g.
`import sparkdl.graph.utils as tfx`.  Thus I kept the names as snake_case.